programs/util.h

```diff
@@ -37,12 +37,17 @@ extern "C" {
 #include <assert.h>
 #include <sys/types.h>    /* stat, utime */
 #include <sys/stat.h>     /* stat */
-#if defined(_MSC_VER)
+#if defined(_WIN32)
 #  include <sys/utime.h>  /* utime */
 #  include <io.h>         /* _chmod */
 #else
 #  include <unistd.h>     /* chown, stat */
+#if PLATFORM_POSIX_VERSION < 200809L
```

Comment:
minor : coding style
since this `#if` is nested inside the `#else`, it would be nice to indent it to the right, alongside its content.

Reply:
I'll indent just the if. I've seen in some other projects ```# if``` works but ```#  if``` does not.

Reply:
Single space vs. double space. GitHub doesn't show it well it seems.